lib/real_savvy/resource.rb

```diff
@@ -1,6 +1,83 @@
-module RealSavvy::Resource
+require 'set'
+require 'real_savvy/attributes'
+require 'real_savvy/links'
 
-end
+class RealSavvy::Resource
```

Comment:
Should you just be using Active Resource instead of trying to create your own Base Resource Class? 

http://api.rubyonrails.org/v3.2.6/classes/ActiveResource/Base.html

Reply:
Don't want the dependencies that come with ActiveResource.